src/lib/support/notifySupportAboutFormSubmissionProblem.ts

```diff
@@ -26,9 +26,9 @@ export async function notifySupportAboutFormSubmissionProblem(
       preferredLanguage: preferredLanguage,
     });
   } catch (error) {
-    logMessage.error(
+    logMessage.info(
```

Comment:
I'm wondering if we would want this to be a `warn` level.  If the error goes completely silent we would never know that a client requires support with a submission.